teamcity-rest-client-impl/src/main/kotlin/org/jetbrains/teamcity/rest/implementation.kt

```diff
@@ -545,6 +545,19 @@ private class BuildImpl(private val bean: BuildBean,
 
         LOG.debug("Artifact '$artifactPath' from build $buildNumber (id:${id.stringId}) downloaded to $output")
     }
+
+    override fun downloadBuildLog(output: File) {
+        LOG.info("Downloading build log from build $buildNumber (id:${id.stringId}) to $output")
+
+        output.parentFile.mkdirs()
+        val response = instance.service.buildLog(id.stringId)
+        val input = response.body.`in`()
```

Comment:
I would extract duplicating code from this function and `downloadAritfact` function into a separate function.

Reply:
Thank you. I updated the PR.